src/Blockcore/Interfaces/INetworkWeight.cs

```diff
@@ -0,0 +1,9 @@
+﻿using NBitcoin;
+
+namespace Blockcore.Interfaces
+{
+    public interface INetworkWeight
+    {
+        double GetPosNetworkWeight();
```

Comment:
Do we call PoS for Pos or PoS in types?

Reply:
I think it follows the rest of the pattern like `PosConsensusFactory`